pkg/kritis/testutil/metadata_mock.go

```diff
@@ -26,10 +26,11 @@ import (
 )
 
 type MockMetadataClient struct {
-	Vulnz           []metadata.Vulnerability
-	PGPAttestations []metadata.PGPAttestation
-	Occ             map[string]string
-	Err             error
+	Vulnz              []metadata.Vulnerability
+	PGPAttestations    []metadata.PGPAttestation
+	KritisAttestations []kritisv1beta1.AttestationAuthority
```

Comment:
Call this 'AttestationAuthorities' or 'AAs' or something such (they are not attestations).